src/www/system_general.php

```diff
@@ -302,9 +302,9 @@
               <td>
                 <input name="domain" type="text" value="<?=$pconfig['domain'];?>" />
                 <div class="hidden" data-for="help_for_domain">
-                  <?=gettext("Do not use 'local' as a domain name. It will cause local hosts running mDNS (avahi, bonjour, etc.) to be unable to resolve local hosts not running mDNS."); ?>
+                  <?=gettext("Do not use 'local' as your internal domain name. It's reserved for and will interfere with mDNS (avahi, bonjour, etc.). Use the special-purpose home.arpa domain instead."); ?>
```

Comment:
Can you use “it is” here. Trying to adhere to technical writing when changing existing texts too.